packages/backend/src/open_payments/receiver/service.ts

```diff
@@ -45,8 +57,30 @@ export async function createReceiverService(
   }
 
   return {
-    get: (url) => getReceiver(deps, url)
+    get: (url) => getReceiver(deps, url),
+    create: (url) => createReceiver(deps, url)
+  }
+}
+
+async function createReceiver(
```

Comment:
Should this allow creating a local incoming payment as well? When we ever want to use this over `incomingPaymentService.create`?

Reply:
The `createReceiver` mutation resolver should support creating local or remote incoming payments.
The decision/attempt to create locally would either need to happen in the resolver or here.
I vote here.

Reply:
Makes sense, I can add that in.=